components/schema-migrator/migrations/director/20210302174853_add_app_template_to_webhooks.down.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+
+ALTER TABLE webhooks
+    DROP CONSTRAINT IF EXISTS webhook_app_id_type_unique,
+    DROP CONSTRAINT IF EXISTS webhook_app_template_id_type_unique,
+    DROP CONSTRAINT IF EXISTS webhook_runtime_id_type_unique,
+    DROP CONSTRAINT IF EXISTS webhook_integration_system_id_type_unique,
+    DROP CONSTRAINT IF EXISTS webhook_owner_id_unique,
+    ALTER COLUMN tenant_id SET NOT NULL,
+    DROP CONSTRAINT IF EXISTS webhooks_app_template_id_fkey,
+    DROP COLUMN IF EXISTS app_template_id;
+
+UPDATE applications
+SET app_template_id = NULL
+WHERE id = id;
+
+ALTER TABLE applications
+    DROP CONSTRAINT IF EXISTS applications_app_template_id_fkey,
```

Comment:
`applications_app_template_id_fkey` is not defined at UP script, and I cannot find it in other SQLs too. Can you please check this.
Probably you should name the constraint in UP script:
```
ADD CONSTRAINT applications_app_template_id_fkey 
   FOREIGN KEY (app_template_id) 
   REFERENCES app_templates(id) 
   ON DELETE SET NULL;
```

Reply:
Good catch - `applications_app_template_id_fkey` was the default name generated for the FK which is added in the UP migration. But it will be more correct to add the name explicitly like you proposed so that we can be sure that the constraint will be named this way rather than expect Postgres to always name it like that.